### ISSUE TITLE
Server rendering failed

### ISSUE THREAD

**Issue by prontiol** [malte-wessel/react-custom-scrollbars#5]
It seems custom scrollbars do not support server rendering, assuming `document` is always available.
Server rendering fails with error: 
```
ReferenceError: document is not defined
      at Object.addStyleSheet [as default] (/node_modules/react-custom-scrollbars/lib/utils/addStyleSheet.js:10:17)
      at [object Object].componentWillMount (/node_modules/react-custom-scrollbars/lib/Scrollbars.js:131:40)
```

**Comment by malte-wessel** [comment]
Thanks for reporting! This will be fixed in the next version!

**Comment by malte-wessel** [comment]
This should be fixed with v2.0.0. Let me know if not.

**Comment by prontiol** [comment]
Nope.
```
  ReferenceError: document is not defined
      at getScrollbarWidth (/node_modules/react-custom-scrollbars/lib/utils/getScrollbarWidth.js:18:9)
      at [object Object].render (/node_modules/react-custom-scrollbars/lib/Scrollbars.js:339:62)
```

**Comment by malte-wessel** [comment]
Sure you installed v2.0.0?

**Comment by prontiol** [comment]
Yep, `npm list react-custom-scrollbars` gives `react-custom-scrollbars@2.0.0 `
But the code looks ok and should not raise this error, strange...
I will investigate this more.

**Comment by prontiol** [comment]
`If (document) { ... }` fails with `ReferenceError: document is not defined` when document is not defined variable.
You should consider using something like `if (typeof document !== 'undefined')`.

**Comment by malte-wessel** [comment]
Should be fixed with #9
